tests/rendering.test.tsx

```diff
@@ -62,18 +73,45 @@ describe("Rendering", () => {
   });
 
   it("[Server] Renders as expected when using `renderToNodeStream`", async () => {
+    const { sketch } = setupTest();
     const nodeStream = renderToNodeStream(<ReactP5Wrapper sketch={sketch} />);
     const content = await unwrapReadableStream(nodeStream);
 
     expect(content).toBe('<div class="react-p5-wrapper"></div>');
   });
 
   it("[Server] Renders as expected when using `renderToStaticNodeStream`", async () => {
+    const { sketch } = setupTest();
     const staticNodeStream = renderToStaticNodeStream(
       <ReactP5Wrapper sketch={sketch} />
     );
     const content = await unwrapReadableStream(staticNodeStream);
 
     expect(content).toBe('<div class="react-p5-wrapper"></div>');
   });
+
+  it("[General] should throw an error when the `sketch` prop is not provided", () => {
+    expect(() => render(<ReactP5Wrapper />)).toThrow(/sketch/);
+  });
+
+  xit("[General] Should not render anything when the `sketch` prop is not provided", () => {
```

Comment:
I was wondering if we skip these tests on purpose?

Reply:
Yes because there is a followup required that is unrelated to the actual PR but this is here as a reminder of sorts 👍🏻